src/camvidlog2/moondream/ai.py

```
@@ -0,0 +1,82 @@
+import time
+from typing import Iterable
```

Ruff (0.12.2)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)